### ISSUE TITLE
broken volume_status

### ISSUE THREAD

**Issue by raingart** [ultrabug/py3status#1393]
broken output **volume_status**

![screenshot from 2018-07-14 17-03-36](https://user-images.githubusercontent.com/13064767/42725167-dd215dd2-8787-11e8-82c6-7f9e2e334a23.png)

py3status v.3.11-1

**Comment by lasers** [comment]
You got an error code. Run `pactl info` in the terminal and see what spits out. Also, if you installed `pulseaudio` recently, restart the system and see if it solves anything.

**Comment by lasers** [comment]
Try adding `command = 'alsa'` to your `volume_status` config if you want `alsa`.

I assume it picked `pactl` because you have it installed (via `libpulse`). Maybe `volume_status` need a better sound server detection.

**Comment by raingart** [comment]
did not work![screenshot from 2018-07-14 17-32-30](https://user-images.githubusercontent.com/13064767/42725427-eda558b2-878b-11e8-98e9-7625777261f8.png)

**Comment by lasers** [comment]
Sorry, I meant `amixer`. Find out why `pactl info` is throwing an error. Maybe you need to install `pulseaudio` too.... When you fix this, then `volume_status` should work okay with `pactl` or if you prefer, use `command = amixer`.

**Comment by raingart** [comment]
thanks @lasers , now it worked. (I myself could have guessed that it was not `alsa` but `amixer`).

>Find out why pactl info is throwing an error.

To be honest, I was always satisfied with the modest possibilities of `alsa` with fewer dependencies. So I'm an adherent of the _sect of alsa lovers_, and against the _blasphemous_ `pulsa`

thank you @lasers

**Comment by lasers** [comment]
@raingart Please leave this open for nice folks in charge. Thanks. :+1: 

Folks in charge, I present you a better sound server detection. If `pactl` is selected automatically, check for `pulseaudio` too. Otherwise, move on to `amixer`. IIRC This issue might be the second one related to `alsa` so I think we should merge this as `libpulse` could be automatically installed in some distros.
```diff
diff --git a/py3status/modules/volume_status.py b/py3status/modules/volume_status.py
index f9eb28f2..62ddd62b 100644
--- a/py3status/modules/volume_status.py
+++ b/py3status/modules/volume_status.py
@@ -304,6 +304,10 @@ class Py3status:
         if not self.command:
             self.command = self.py3.check_commands(
                 ['pamixer', 'pactl', 'amixer'])
+            # pactl may be automatically installed, check for pulseaudio too
+            if self.command == 'pactl':
+                if not self.py3.check_commands('pulseaudio'):
+                    self.command = self.py3.check_commands('amixer')
         elif self.command not in ['amixer', 'pamixer', 'pactl']:
             raise Exception(STRING_ERROR % self.command)
         elif not self.py3.check_commands(self.command):

```

**Comment by ultrabug** [comment]
fixed by #1400 thx!
